README.md

```diff
@@ -2,6 +2,8 @@
 
 A web application that shows various lockdown states for countries across the world helping people to find out if they can travel and at what restrictions.
 
+Required Components: Node Version 10.0, preact v10
```

Comment:
Preact is not a component that's required to be installed, it's a dependency among many others. Just listing `npm install` should suffice

Reply:
@LarsDenBakker : Ok we don't have to add description of what it does?